newIDE/app/src/AssetStore/AssetPackTiles.js

```diff
@@ -0,0 +1,366 @@
+// @flow
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import {
+  type PublicAssetPack,
+  type AssetShortHeader,
+} from '../Utils/GDevelopServices/Asset';
+import { type PrivateAssetPackListingData } from '../Utils/GDevelopServices/Shop';
+import { GridListTile, createStyles, makeStyles } from '@material-ui/core';
+import { shouldValidate } from '../UI/KeyboardShortcuts/InteractionKeys';
+import Paper from '../UI/Paper';
+import { CorsAwareImage } from '../UI/CorsAwareImage';
+import { textEllipsisStyle } from '../UI/TextEllipsis';
+import { Column, Line } from '../UI/Grid';
+import Text from '../UI/Text';
+import { Trans } from '@lingui/macro';
+import {
+  PrivateAssetPackPriceTag,
+  formatPrivateAssetPackPrice,
+} from './PrivateAssets/PrivateAssetPackPriceTag';
+import { AssetCard } from './AssetCard';
+import FolderIcon from '../UI/CustomSvgIcons/Folder';
+import FlatButton from '../UI/FlatButton';
+
+const capitalize = (str: string) => {
```

Comment:
The same function is defined in FiltersChooser. I think you can put the function in StringHelpers

Reply:
yes nice